rke/k8s_rke_system_images.go

```diff
@@ -5998,10 +5998,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
-		// Enabled out of band post v2.5.11
-		"v1.20.13-rancher1-1": {
+		// Enabled in v2.5.12
```

Comment:
I've been told we will be doing OOB KDM releases for both 2.5 and 2.6, so this should be `Enabled out of band post v2.5.11”`

Reply:
It's fixed now.

Reply:
It's fixed now.